test/helpers/browser.js

```diff
@@ -7,6 +7,7 @@ var vm = require('vm');
 
 function createContext() {
   var context = {
+    self: this,
```

Comment:
so users wouldnt have had this problem to begin with right? most browsers have `self` as the `window` object anyways, so perhaps the change in [formData](https://github.com/form-data/form-data/commit/2c0a54532c22da345e85afd559a9c6a14ec8155b) does not actually affect the end users themselves?